lib/rototiller/task/params/command.rb

```diff
@@ -161,6 +161,9 @@ def run
         @result = Result.new
         @result.output = ''
 
+        # Print out the command we're running so users can see what was executed
+        puts self.to_str
```

Comment:
Wow where we not doing this?!?

Reply:
We do this with {{--verbose}} turned on in the Command loop in rototiller_task:
https://github.com/puppetlabs/rototiller/blob/master/lib/rototiller/task/rototiller_task.rb#L108
if we think we want this all the time, we can remove the conditional.  we should probably change its color, or make it clear we're logging it, and it's not the shell.